tools/cp-caps/README.md

```diff
@@ -0,0 +1,130 @@
+# RCP Capabilities Test
+
+This test is used for testing RCP capabilities.
+
+## Test Topology
+
+```
+                    +-------+
+    +---------------|  PC   |----------------+
+    |               +-------+                |
+    | ADB/SSH                                | ADB/SSH/SERIAL
+    |                                        |
++-------+                           +------------------+
+|  DUT  |<-----------Thread-------->| Reference Device |
++-------+                           +------------------+
+
+```
+
+- PC : The computer to run the test script.
+- DUT : The device under test.
+- Reference Device : The device that supports all tested features.
+
+### Python Dependences
+
+Before running the test script on PC, testers should install dependences first.
+
+```bash
+$ cd tools/otci
+$ pip install build
+$ python -m build
+$ pip install dist/otci_openthread-0.0.1-py3-none-any.whl
+$ pip install adb-shell
+$ pip install adb-shell[usb]
```

Comment:
nit: probably you can make a `requirements.txt` so the user don't have to manually install them one by one

Reply:
I have added the `requirements.txt`.